### PR TITLE
[thread-cert] refactor case 7.1.3 using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_7_1_03_BorderRouterAsLeader.py [hunk 1]
```diff
@@ -96,91 +118,123 @@ def test(self):
...
+                must_next()
+            pkts.filter_wpan_src64(LEADER).\
+                filter_wpan_dst64(MED).\
+                filter_mle_cmd(MLE_CHILD_UPDATE_RESPONSE).\
+                filter(lambda p: {
+                                  SOURCE_ADDRESS_TLV,
+                                  MODE_TLV,
+                                  ADDRESS_REGISTRATION_TLV
+                                 } < set(p.mle.tlv.type) and\
+                       set(p.mle.tlv.addr_reg_iid) < set(_pkt.mle.tlv.addr_reg_iid)
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_7_1_03_BorderRouterAsLeader.py:185]: further validate the addresses echoed back?
The current check can pass even when no address is echoed back.

File: tests/scripts/thread-cert/Cert_7_1_03_BorderRouterAsLeader.py [hunk 2]
```diff
@@ -96,91 +118,123 @@ def test(self):
...
+            must_next()
+        pkts.filter_wpan_src64(LEADER).\
+            filter_wpan_dst64(SED).\
+            filter_mle_cmd(MLE_CHILD_UPDATE_RESPONSE).\
+            filter(lambda p: {
+                              SOURCE_ADDRESS_TLV,
+                              MODE_TLV,
+                              ADDRESS_REGISTRATION_TLV
+                             } < set(p.mle.tlv.type) and\
+                   set(p.mle.tlv.addr_reg_iid) < set(_pkt.mle.tlv.addr_reg_iid)
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_7_1_03_BorderRouterAsLeader.py:235]: ditto

File: tests/scripts/thread-cert/Cert_7_1_03_BorderRouterAsLeader.py (hunk 1 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_7_1_03_BorderRouterAsLeader.py]: added nullField check

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

**Review by jwhui — APPROVED** [review]

**Pull request merged**